### ISSUE TITLE
Show on Identity Server home page link to all clients

### ISSUE THREAD

**Issue by aiscrim** [skoruba/IdentityServer4.Admin#467]
I think it would be very practical for users to be able to access the Identity Server and, after being logged in, being presented all the clients, or - it would be even better - all the clients they have access to.
I can open a (low priority) PR for that, but I'd like to ask you for your approval, and also have some suggestion: how would you read the list of clients? Using the Admin.Api? Reading them somehow directly from the db? Adding them to a new appsettings section?

**Comment by skoruba** [comment]
What do you mean with presenting of all available clients? What do you want to show to the users - configuration of clients?

**Comment by aiscrim** [comment]
No, sorry for not being clear enough, I meant the links to the clients themselves, like the way it’s done for the Admin client, but generalized to every client (so instead of putting the links in the top menu I’d put them in the home page).
That way users could be given just the link to the Identity Server and from there they would be able to navigate to all the applications they have access to. Does it make sense?

**Comment by HrDahl** [comment]
I would say it only makes sense if I am interested in the other Clients (projects). 

E.g. for our specific project we do actually have a list of links to all our other clients on the landing page of the IdP but that is primarily because all the other clients (projects) are developed inhouse and we want to make users aware about all the different possibilities our other products/projects can offer them.

**Comment by aiscrim** [comment]
@HrDahl yes, this is exactly my use case too. If you already implemented it, could you share that code with us? Then we can decide if leaving it by default or upon the setting of an appsetting parameter if you think that not everybody could be interested in it. My point of view is that if an application is integrated then it is of interest to the end user (at least if the user has access to it), regardless of it having been developed inhouse or not.

**Comment by HrDahl** [comment]
@aiscrim I see some cases where I would not be interested in getting links/information about certain Clients, so making it optional via an appsettings parameter seems like a good idea.
Let me see if I can find some time to make a PR with source code.

**Comment by skoruba** [comment]
@aiscrim - if you want - you can mail me your contact where we can discuss details

**Comment by aiscrim** [comment]
Ok, you got mail! :-)
